requirements/build.in

```diff
@@ -0,0 +1,6 @@
+-r base.in
+setuptools_scm>=6.4.0
+dulwich>=0.20.30
```

Comment:
You're declaring this as a _top_-level dependency, but I can't find any code that's actually using it. Where/how does `dulwich` come into play? If this wasn't an oversight, consider adding a comment here.

Reply:
the "build" dependencies are only used by the release-maintainer; dulwich is used in the new `misc/build_helpers` script .. which apparently I forgot to commit ;) so thanks